main/src/main/java/com/github/topi314/lavasrc/yandexmusic/YandexMusicSourceManager.java

```diff
@@ -26,9 +26,11 @@
 import java.util.function.Consumer;
 import java.util.function.Function;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 public class YandexMusicSourceManager implements AudioSourceManager, HttpConfigurable {
 	public static final Pattern URL_PATTERN = Pattern.compile("(https?://)?music\\.yandex\\.(ru|com)/(?<type1>artist|album)/(?<identifier>[0-9]+)/?((?<type2>track/)(?<identifier2>[0-9]+)/?)?");
+	public static final Pattern URL_TRACK_PATTERN = Pattern.compile("(https?://)?music\\.yandex\\.(ru|com)/(?<type1>track)/(?<identifier>[0-9]+)");
```

Comment:
```suggestion
	public static final Pattern URL_TRACK_PATTERN = Pattern.compile("(https?://)?music\\.yandex\\.(ru|com)/(?<type1>track)/(?<identifier>[0-9a-z-]+)");
```
Custom tracks have a different identifier pattern.